### PR TITLE
UHM-4184 Updated PIH colors

### PR DESCRIPTION
**Pull request by lnball**
Working space until I get it reviewed by John Ra...  Keep tweaking...

### CODE REVIEW

File: configuration/pih/styles/global/zl.css [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+	color: #0f5149;
+}
+
+/*
+ui.select li.selected {
+	color: white;
+	background-color: #0f5149;
+}
+*/
+
```
Comment (brandones) [inline configuration/pih/styles/global/zl.css:62]:
```suggestion
```

File: configuration/pih/styles/global/zl.css [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+.new-patient-header .identifiers span.add-id a,
+.patient-header .identifiers span.add-id a:link,
+.patient-header .identifiers span.add-id a:visited {
+	color: #3AB4B1;
+}
+
+.button.confirm, button.confirm,
+input[type="submit"].confirm, input[type="button"].confirm, input[type="submit"].confirm,
+a.button.confirm {
```
Comment (brandones) [inline configuration/pih/styles/global/zl.css:71]:
```suggestion
.button.confirm,
button.confirm,
input[type="submit"].confirm,
input[type="button"].confirm,
input[type="submit"].confirm,
a.button.confirm {
```

File: configuration/pih/styles/global/zl.css [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+	background: #0F5149;
+	border-color: #0f5149;
+}
+
+.button, button, input[type="submit"], input[type="button"], input[type="submit"], a.button {
```
Comment (brandones) [inline configuration/pih/styles/global/zl.css:76]:
```suggestion
.button, button, input[type="submit"], input[type="button"], a.button {
```

**Comment by brandones** [comment]
Made some suggestions but generally looks good! Can you post a screenshot?

**Comment by brandones** [comment]
Oh, just saw the screenshots on the ticket. Looks good! Up to you whether you want to wait for John Ra's feedback or address it in a separate PR later. Thanks for your work on this.

Also, sorry I wasn't clear about this in our call—since it's clear this is very easy for sites to customize, I'm totally open to this going into config-pihemr. If individual sites don't like it they can add their own overrides. But I think it's fine PIH branding, and suspect that at least Mexico will also be happier with it.

**Comment by lnball** [comment]
I'll wait to see if John responds tomorrow and then just go with it.  He already made suggestions, so I followed his lead.  Thanks Brandon
________________________________

**Comment by mogoodrich** [comment]
fyi I know we are doing a release for KGH soon... are we fine with surprising them with a color change?

**Comment by lnball** [comment]
No surprise.  It’s only ZL config unless others want it.  But we’ll want to let ZL know before their next release.

Sent from my iPhone

On Mar 29, 2021, at 6:40 PM, Mark Goodrich ***@***.***> wrote:

﻿

fyi I know we are doing a release for KGH soon... are we fine with surprising them with a color change?

**Comment by mogoodrich** [comment]
Ah, right, I missed that...

**Pull request merged**